ocaml/xapi/xapi_sr_operations.ml

```diff
@@ -86,6 +94,9 @@ let valid_operations ~__context ?op record _ref' : table =
         if Hashtbl.find table op = None
         then Hashtbl.replace table op (Some(code, params))) ops in
 
+  if Helpers.rolling_upgrade_in_progress ~__context then begin
+    set_errors Api_errors.not_supported_during_upgrade [] disallowed_during_rpu
+  end;
```

Comment:
I reckon you don't need `begin ... end` here, unless that's just to make it look clearer?

Reply:
yep, it's to make it look clearer.